src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -12,6 +12,7 @@ import org.rust.ide.icons.addVisibilityIcon
 import org.rust.lang.core.psi.*
 import org.rust.lang.core.psi.ext.*
 import org.rust.lang.core.types.type
+import kotlin.collections.ArrayList
```

Comment:
For me this shows a "unresolved reference" error in IntelliJ but compiles fine in Gradle.
Maybe you can use `mutableListOf<String>()` below which should work the same way.